### ISSUE TITLE
Header - logo SVG with no CSS

### ISSUE THREAD

**Issue by jtart** [bbc/simorgh#378]
If the user doesn’t have CSS (it either hasn't loaded or they have chosen to disable it), it would be awesome if the user can still see the brand logo SVG i.e. a white SVG cannot be seen on a white background. 

The following example code demonstrates how you can display an SVG in black for users without CSS, by specifying an inline fill colour (black) for the SVG which is available if the user doesn’t have CSS (which works on a white background), the colour of the SVG is then changed to white with CSS which can then be seen on the CSS coloured background. An example of how to implement this can be seen here http://2bitstudio.co.uk/code-tests/article/svg-brand.html

**Additional context - High contrast mode on PC**
It's important that the brand logo SVG works with default and custom colour themes (at a minimum with CSS enabled), ensure you test this in development.

**Comment by ChrisBAshton** [comment]
Related to #860.

**Comment by ChrisBAshton** [comment]
Fixed by https://github.com/BBC-News/psammead/pull/253.
